### ISSUE TITLE
Accessibility Feedback for Search & Discovery-Search UI Consistency Pattern for Global Search Pages

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#26323]
# Accessibility Feedback for Search & Discovery-Search UI Consistency Pattern for Global Search Pages

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

**Comment by joshkimux** [comment]
Potential bugs caught:
<img width="1474" alt="Screen Shot 2021-06-21 at 1 28 56 PM" src="https://user-images.githubusercontent.com/14154792/122803618-d413eb00-d294-11eb-8641-3b3897eb8e20.png">

**Comment by joshkimux** [comment]
Question for the team @denisecoveyduc , is the "Back to top" on the Find a VA Form search results page within scope of this review? It does not exist on the other search results pages, so I'm curious if that's an inconsistency that should be flagged and audited for accessibility.

**Comment by mgleo** [comment]
Correct, it would be a Decision Tools' teams feature.

**Comment by MarciMcGuireGCIO** [comment]
@joshkimux @denisecoveyduc hmm... I didn't think Find a VA Form had a "back to top", so I'm looping in @zacharymorel to get his take on it.

**Comment by zacharymorel** [comment]
Good morning all, 

**Back to top** is a global site wide component that's apart of the design system components, it appears on quite a few static pages. 
If there are any issues with it, VSP's design system team needs to be notified. You can do that by filling out a request in the vfs-platform-support channel.

**Comment by denisecoveyduc** [comment]
Thanks @MarciMcGuireGCIO & @zacharymorel 

@joshkimux Can you confirm, is there an accessibility issue with the "Back to Top" link located at the bottom of the Find a Form page? 

fyi: @mgleo

**Comment by joshkimux** [comment]
@denisecoveyduc I'll be writing a separate ticket addressing issues as Zach pointed out this is a global site wide component.
The main issue I'd like to test more is its stickiness, but that's not something that needs to be handled here! 🙇

**Comment by denisecoveyduc** [comment]
@joshkimux with the closure of  #26352 I believe we are good to close this epic now as well.

**Comment by joshkimux** [comment]
@denisecoveyduc Yes! Great work from the team! 🥳
